xml/deploy_cephadm.xml

```diff
@@ -701,131 +701,293 @@ o- time_server ................................................ [enabled]
-    </para>
-    <procedure>
-     <step>
       <para>
-       Configure the URL of the local registry:
+       On the registry node, use the following command to ensure that the port
+       is free:
       </para>
-<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/registry set <replaceable>REGISTRY_URL</replaceable></screen>
-     </step>
-     <step>
+<screen>ss -tulpn | grep :5000</screen>
       <para>
-       Configure the user name and password to access the local registry:
+       If other processes (such as <literal>iscsi-tcmu</literal>) are already
+       listening on port 5000, determine another free port which can be used to
+       map to port 5000 in the registry container.
       </para>
-<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/username set <replaceable>REGISTRY_USERNAME</replaceable></screen>
-<screen>&prompt.cephuser;ceph-salt config /containers/registry_auth/password set <replaceable>REGISTRY_PASSWORD</replaceable></screen>
-     </step>
-     <step>
+     </tip>
+     <procedure>
+      <title>Creating the local registry</title>
+      <step>
+       <para>
+        Verify that the <package>Containers Module</package> extension is
+        enabled:
+       </para>
+<screen>
+&prompt.user;SUSEConnect --list-extensions | grep -A2 "Containers Module"
+Containers Module 15 SP2 x86_64 (Activated)
+</screen>
+      </step>
+      <step>
+       <para>
+        Verify that the following packages are installed:
+        <package>apache2-utils</package> (if enabling a secure registry),
+        <package>cni</package>, <package>cni-plugins</package>,
+        <package>podman</package>, <package>podman-cni-config</package>, and
+        <package>skopeo</package>.
+       </para>
+      </step>
+      <step>
+       <para>
+        Gather the following information:
+       </para>
+       <itemizedlist>
+        <listitem>
+         <para>
+          Fully qualified domain name of the registry host
+          (<option>REG_HOST_FQDN</option>).
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+          An available port number used to map to the registry container port
+          of 5000 (<option>REG_HOST_PORT</option>).
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+          Whether the registry will be secure or insecure
+          (<option>insecure=[true|false]</option>).
+         </para>
+        </listitem>
+       </itemizedlist>
+      </step>
+      <step>
+       <para>
+        Configure the registry by using &cephsalt;:
+       </para>
+<screen>
+&prompt.cephuser;ceph-salt config containers/registries_conf enable
+&prompt.cephuser;ceph-salt config containers/registries_conf/registries \
+ add prefix=<option>REG_HOST_FQDN</option> insecure=[true|false] \
+ location=<option>REG_HOST_PORT</option>:5000
+&prompt.cephuser;ceph-salt apply --non-interactive
+&prompt.cephuser;salt '*' cmd.shell "systemctl restart podman"
```

Comment:
I slightly dubious about this `systemctl restart podman` command. AFAIK we don't usually run the podman service, so I'm not sure why it's here.